converters/toZigbee.js

```diff
@@ -3660,6 +3661,146 @@ const converters = {
             await sendTuyaDataPointValue(entity, common.TuyaDataPoints.saswellTempCalibration, value);
         },
     },
+    silvercrest_smart_led_string: {
+        key: ['silvercrest_smart_led_string', 'color', 'brightness', 'scene'],
+        convertSet: async (entity, key, value, meta) => {
+            const scale = (value, valueMin, valueMax, min, max) => {
+                return min + ((max-min) / (valueMax - valueMin)) * value;
+            };
+
+            const make4sizedString = (v) => {
+                if (v.length >= 4) return v;
+                else if (v.length === 3) return '0'+v;
+                else if (v.length === 2) return '00'+v;
+                else if (v.length === 1) return '000'+v;
+                else return '0000';
+            };
+
+            if (key === 'scene') {
+                await sendTuyaDataPointEnum(entity, common.TuyaDataPoints.silvercrestChangeMode, common.silvercrestModes.scene);
+
+                let data = [];
+                const scene = common.silvercrestEffects[value.scene];
+                data = data.concat(utils.convertStringToHexArray(scene));
+                let speed = Math.round(scale(value.speed, 0, 100, 0, 64));
+
+                // Max speed what the gateways sends is 64.
+                if (speed > 64) {
+                    speed = 64;
+                }
+
+                // Make it a string and attach a leading zero (0x30)
+                let speedString = String(speed);
+                if (speedString.length === 1) {
+                    speedString = '0' + speedString;
+                }
+                if (!speedString) {
+                    speedString = '00';
+                }
+
+                data = data.concat(utils.convertStringToHexArray(speedString));
+                let colors = value.colors;
```

Comment:
What does `colors` do, is it possible to trigger an effect with a specific color?

Reply:
Yes. Effects take zero to 6 colors (dependend on the effect).
For example: you can do an effect "Snake" which cycles a color through your christmas tree as a snake. The user can choose 1 - 3 colors for this effect.

Reply:
Can you write some documentation for this (with payloads to send)? Otherwise it will be hard to figure out what to send.  Create a pr for https://github.com/Koenkk/zigbee2mqtt.io/tree/master/docs/devices adding a file called `HG06467.md` this file only has to contain how to trigger the scenes. e.g. (the other stuff will automatically be generated later).

### Triggering scenes
Overview of possible scenes: ......
To trigger them send the following payload to `zigbee2mqtt/FRIENDLY_NAME/set` `{"scene": {....}}`.

Reply:
Will do.